README.rst

```diff
@@ -54,7 +54,7 @@ within the CMake source directory or any other build directory of your
 choice. Once this has finished successfully, run ``make`` and
 ``make install``.  In summary::
 
- $ ./bootstrap && make && make install
+ $ ./bootstrap && gmake && gmake install
```

Comment:
@gecube What was the reason for this change? It was suggested to change this back to `make`.